tests/tests_pytorch/trainer/logging_/test_eval_loop_logging.py

```diff
@@ -527,57 +527,47 @@ def test_step(self, batch, batch_idx):
     trainer = Trainer(
         default_root_dir=tmpdir,
         logger=TensorBoardLogger(tmpdir),
-        limit_train_batches=2,
+        limit_train_batches=1,
```

Comment:
I would keep at least two so we test iteration

Reply:
This is an evaluation logging tests. There's a thousand other tests that test more than 1 batch :joy: